src/mpl2/src/mpl.tcl

```diff
@@ -31,34 +31,82 @@
 ## OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 ############################################################################
 
-sta::define_cmd_args "rtl_macro_placer" { -config_file config_file \
-                                         [-report_directory report_file] \
-                                         -report_file report_file \
+sta::define_cmd_args "rtl_macro_placer" { -report_directory report_dir \
+                                         [-config_file config_file]
```

Comment:
Let's remove -config_file from here so that it doesn't appear in the help.  It will still be accessible below to developers.

Reply:
removed -config_file from the cmd_args